configure.ac

```diff
@@ -272,9 +272,9 @@ AM_CONDITIONAL(LIBELF,[test "$WITH_LIBELF" = yes])
 # Select crypto library
 AC_ARG_WITH(crypto,
             [AC_HELP_STRING([--with-crypto=CRYPTO_LIB],
-                            [The cryptographic library to use (openssl|libgcrypt). The default is libgcrypt.])
+                            [The cryptographic library to use (sequoia|openssl|libgcrypt). The default is sequoia.])
                             ],[],
-                            [with_crypto=libgcrypt])
+                            [with_crypto=sequoia])
```

Comment:
I don't think we're ready to swap the default just yet, and even the commit message merely speaks of adding an alternative.

Other than that, it all looks remarkably straightforward as all the hard stuff happens elsewhere.

Reply:
> I don't think we're ready to swap the default just yet, and even the commit message merely speaks of adding an alternative.

I'll revert that.

> Other than that, it all looks remarkably straightforward as all the hard stuff happens elsewhere.

Yup.  Happily even that (rpm-sequoia) is not that complex:

```
$ loc .
--------------------------------------------------------------------------------
 Language             Files        Lines        Blank      Comment         Code
--------------------------------------------------------------------------------
 Rust                     6         1991          206          295         1490
...
```